### ISSUE TITLE
Fix tool tip menu on persistent navigation example

### ISSUE THREAD

**Issue by halocline** [grommet/hpe-design-system#1148]
Reported by @gautams

"Hi team, the tool tip Menu labels on the Persistent Navigation template seem to be violating the WCAG success criterion for content on Hover/focus. "If pointer hover can trigger the additional content, then the pointer can be moved over the additional content without the additional content disappearing;" Is there some way to make this possible without using timers and such?"

**Comment by taysea** [comment]
I've compiled a few resources around accessible tooltips:
- https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html
- https://diginclusion.com/industry-updates/wcag-2-1-what-is-content-on-hover-or-focus/#:~:text=If%20pointer%20hover%20can%20trigger,information%20is%20no%20longer%20valid.

However, before diving too deep here, I know that @vavalos5, @ShimiSun, and others had some discussions around tool tips (with potential add to grommet), so I'd like to touch base with outcomes of that before moving forward here. I assume their work can be leveraged to provide direction to fix/improve this.

**Comment by taysea** [comment]
@britt6612 Can you confirm -- did we remove the "Persistent Navigation" example? If so, I think we can close this issue.

**Comment by britt6612** [comment]
> Persistent Navigation

@taysea it was removed
https://design-system.hpe.design/templates/navigation?q=nav
